<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutEvenementWindow</class>
 <widget class="QWidget" name="AjoutEvenementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un événement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="selectionContactLabel">
         <property name="text">
          <string>Sélectionner un contact : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="contactComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="eventRadionButton">
         <property name="text">
          <string>Ajouter un événement existant</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">GroupeInteraction</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="eventComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="newEventRadionButton">
         <property name="text">
          <string>Ajouter un nouvel événement </string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">GroupeInteraction</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="newEventlineEdit">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="validButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="GroupeInteraction"/>
 </buttongroups>
</ui>
